docker/runtime/Dockerfile.pytorch

```diff
@@ -2,7 +2,8 @@ ARG BASEIMAGE=nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
 FROM ${BASEIMAGE}
 
 ADD ./docker/scripts /install/scripts
-RUN bash /install/scripts/find-fastest-apt.sh
+RUN bash /install/scripts/find-fastest-apt.sh \
```

Comment:
Just curious that any apt source will faster than that inside alicloud VPC?

Reply:
the host outside of alicloud ECS can not connect to alicloud apt source.